quinn-udp/src/windows.rs

```diff
@@ -154,4 +369,98 @@ impl UdpSocketState {
     }
 }
 
+fn set_socket_option(
+    socket: &impl AsRawSocket,
+    level: i32,
+    name: i32,
+    value: u32,
+) -> Result<(), io::Error> {
+    let rc = unsafe {
+        WinSock::setsockopt(
+            socket.as_raw_socket() as usize,
+            level,
+            name,
+            &value as *const _ as _,
+            mem::size_of_val(&value) as _,
+        )
+    };
+
+    match rc == 0 {
+        true => Ok(()),
+        false => Err(io::Error::last_os_error()),
+    }
+}
+
 pub(crate) const BATCH_SIZE: usize = 1;
+// Enough to store max(IP_PKTINFO + IP_ECN, IPV6_PKTINFO + IPV6_ECN) + UDP_SEND_MSG_SIZE bytes (header + data) and some extra margin
```

Comment:
`max(UDP_SEND_MSG_SIZE, UDP_COALESCED_INFO)`, right?

Reply:
Yes, good catch, fixing it